atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/util/AtlasDbUtils.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.workload.util;
+
+import com.google.common.primitives.Ints;
+import com.palantir.atlasdb.AtlasDbConstants;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.protos.generated.TableMetadataPersistence;
+import com.palantir.atlasdb.table.description.ColumnMetadataDescription;
+import com.palantir.atlasdb.table.description.NameMetadataDescription;
+import com.palantir.atlasdb.table.description.TableMetadata;
+import com.palantir.atlasdb.transaction.api.ConflictHandler;
+import com.palantir.atlasdb.workload.store.WorkloadCell;
+
+public final class AtlasDbUtils {
+
+    private AtlasDbUtils() {}
+
+    public static Cell toAtlasCell(WorkloadCell cell) {
+        return Cell.create(Ints.toByteArray(cell.key()), Ints.toByteArray(cell.column()));
```

Comment:
Hmm. I guess this is fine especially if we want to support range scans, though this is going to hotspot I think - let's make sure we're ok with that?

Reply:
Yeah, so I had a couple of thoughts here why this _should_ be ok: (1) the keys _should_ be random, so in theory we should have perfect distribution, (2) I think for now, it's highly unlikely we'll have >3 node. 

That said, totally happy to flip the bits around, i.e. little endian.

Reply:
Ah, maybe for now let's not use little endian since we do want range scans!

I figured in our use case we probably have RF = number of nodes, though was thinking @zpear's use case might become relevant later on and for that we might not want to be too tied to that assumption. But it's something we could change later on anyway